app/screens/settings-screen/account-screen.tsx

```diff
@@ -450,6 +502,22 @@ export const AccountScreen = () => {
     },
   ]
 
+  if (beta) {
```

Comment:
I think in other places to remove a setting we set `hidden: true` instead of dynamically adding it to the list.

Reply:
sure thing